<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="9" passed="3" failed="6" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-29T19:04:59 IST" name="Sauce Demo Products" finished-at="2024-06-29T19:05:40 IST" duration-ms="41050">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner1@7b2bbc3]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner2@a1153bc]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner@7c7b252e]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-06-29T19:04:59 IST" name="Login Page Test" finished-at="2024-06-29T19:05:40 IST" duration-ms="41050">
      <class name="com.qa.runner.TestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:com.qa.runner.TestRunner@7c7b252e]" started-at="2024-06-29T19:04:59 IST" name="setUpClass" finished-at="2024-06-29T19:04:59 IST" duration-ms="853" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@69fb6037]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner@7c7b252e]" started-at="2024-06-29T19:04:59 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:20 IST" duration-ms="20407" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["Unsuccessful login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner@7c7b252e]" started-at="2024-06-29T19:04:59 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:00 IST" duration-ms="183" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successful login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at com.google.gson.internal.LinkedTreeMap.replaceInParent(LinkedTreeMap.java:318)
at com.google.gson.internal.LinkedTreeMap.rotateLeft(LinkedTreeMap.java:408)
at com.google.gson.internal.LinkedTreeMap.rebalance(LinkedTreeMap.java:349)
at com.google.gson.internal.LinkedTreeMap.removeInternal(LinkedTreeMap.java:294)
at com.google.gson.internal.LinkedTreeMap.removeInternal(LinkedTreeMap.java:261)
at com.google.gson.internal.LinkedTreeMap.removeInternalByKey(LinkedTreeMap.java:302)
at com.google.gson.internal.LinkedTreeMap.remove(LinkedTreeMap.java:134)
at com.aventstack.extentreports.gherkin.GherkinDialect.<init>(GherkinDialect.java:26)
at com.aventstack.extentreports.gherkin.GherkinDialectManager.setLanguage(GherkinDialectManager.java:76)
at com.aventstack.extentreports.ExtentReports.setGherkinDialect(ExtentReports.java:446)
at com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter.createFeature(ExtentCucumberAdapter.java:314)
at com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter.handleStartOfFeature(ExtentCucumberAdapter.java:307)
at com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter.handleTestCaseStarted(ExtentCucumberAdapter.java:161)
at com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter.access$100(ExtentCucumberAdapter.java:63)
at com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter$3.receive(ExtentCucumberAdapter.java:107)
at com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter$3.receive(ExtentCucumberAdapter.java:104)
at io.cucumber.core.eventbus.AbstractEventPublisher.send(AbstractEventPublisher.java:51)
at io.cucumber.core.eventbus.AbstractEventBus.send(AbstractEventBus.java:12)
at io.cucumber.core.runtime.SynchronizedEventBus.send(SynchronizedEventBus.java:47)
at io.cucumber.core.runtime.ThreadLocalRunnerSupplier$LocalEventBus.send(ThreadLocalRunnerSupplier.java:62)
at io.cucumber.core.runner.TestCase.emitTestCaseStarted(TestCase.java:206)
at io.cucumber.core.runner.TestCase.run(TestCase.java:72)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:75)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:137)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:137)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:81)
at org.testng.internal.thread.Async$AsyncTask.run(Async.java:34)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner@7c7b252e]" started-at="2024-06-29T19:04:59 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:20 IST" duration-ms="20258" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Unsuccessful login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Epic sadface: Username and password do not match any user in this srevice] but found [Epic sadface: Username and password do not match any user in this service]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Epic sadface: Username and password do not match any user in this srevice] but found [Epic sadface: Username and password do not match any user in this service]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.qa.stepdefinitions.LoginPageDefinition.user_should_display_error_message_as(LoginPageDefinition.java:70)
at âœ½.User should display error message as "Epic sadface: Username and password do not match any user in this srevice"(file:///C:/Users/shreenethira/eclipse-workspace/CucumberTest/src/test/java/com/qa/features/LoginPage.feature:35)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:com.qa.runner.TestRunner@7c7b252e]" started-at="2024-06-29T19:05:20 IST" name="tearDownClass" finished-at="2024-06-29T19:05:27 IST" duration-ms="7423" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.qa.runner.TestRunner -->
      <class name="com.qa.runner.TestRunner1">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:com.qa.runner.TestRunner1@7b2bbc3]" started-at="2024-06-29T19:04:59 IST" name="setUpClass" finished-at="2024-06-29T19:04:59 IST" duration-ms="852" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@69fb6037]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner1@7b2bbc3]" started-at="2024-06-29T19:04:59 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:20 IST" duration-ms="20034" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["Unsuccessful login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner1@7b2bbc3]" started-at="2024-06-29T19:04:59 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:20 IST" duration-ms="20627" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successful login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Product] but found [Products]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Product] but found [Products]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.qa.stepdefinitions.LoginPageDefinition.user_should_land_on_products_page(LoginPageDefinition.java:57)
at âœ½.User should land on Products Page(file:///C:/Users/shreenethira/eclipse-workspace/CucumberTest/src/test/java/com/qa/features/LoginPage.feature:29)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner1@7b2bbc3]" started-at="2024-06-29T19:04:59 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:20 IST" duration-ms="20491" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Unsuccessful login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Epic sadface: Username and password do not match any user in this srevice] but found [Epic sadface: Username and password do not match any user in this service]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Epic sadface: Username and password do not match any user in this srevice] but found [Epic sadface: Username and password do not match any user in this service]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.qa.stepdefinitions.LoginPageDefinition.user_should_display_error_message_as(LoginPageDefinition.java:70)
at âœ½.User should display error message as "Epic sadface: Username and password do not match any user in this srevice"(file:///C:/Users/shreenethira/eclipse-workspace/CucumberTest/src/test/java/com/qa/features/LoginPage.feature:35)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:com.qa.runner.TestRunner1@7b2bbc3]" started-at="2024-06-29T19:05:20 IST" name="tearDownClass" finished-at="2024-06-29T19:05:28 IST" duration-ms="7558" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.qa.runner.TestRunner1 -->
      <class name="com.qa.runner.TestRunner2">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:com.qa.runner.TestRunner2@a1153bc]" started-at="2024-06-29T19:05:27 IST" name="setUpClass" finished-at="2024-06-29T19:05:27 IST" duration-ms="19" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@69fb6037]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner2@a1153bc]" started-at="2024-06-29T19:05:27 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:37 IST" duration-ms="9203" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["Unsuccessful login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner2@a1153bc]" started-at="2024-06-29T19:05:27 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:37 IST" duration-ms="9202" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successful login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Product] but found [Products]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Product] but found [Products]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.qa.stepdefinitions.LoginPageDefinition.user_should_land_on_products_page(LoginPageDefinition.java:57)
at âœ½.User should land on Products Page(file:///C:/Users/shreenethira/eclipse-workspace/CucumberTest/src/test/java/com/qa/features/LoginPage.feature:29)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.qa.runner.TestRunner2@a1153bc]" started-at="2024-06-29T19:05:27 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-29T19:05:36 IST" duration-ms="8874" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Unsuccessful login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Page"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Epic sadface: Username and password do not match any user in this srevice] but found [Epic sadface: Username and password do not match any user in this service]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Epic sadface: Username and password do not match any user in this srevice] but found [Epic sadface: Username and password do not match any user in this service]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.qa.stepdefinitions.LoginPageDefinition.user_should_display_error_message_as(LoginPageDefinition.java:70)
at âœ½.User should display error message as "Epic sadface: Username and password do not match any user in this srevice"(file:///C:/Users/shreenethira/eclipse-workspace/CucumberTest/src/test/java/com/qa/features/LoginPage.feature:35)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:com.qa.runner.TestRunner2@a1153bc]" started-at="2024-06-29T19:05:37 IST" name="tearDownClass" finished-at="2024-06-29T19:05:40 IST" duration-ms="3092" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.qa.runner.TestRunner2 -->
    </test> <!-- Login Page Test -->
  </suite> <!-- Sauce Demo Products -->
</testng-results>
